.github/workflows/lint.yml

```diff
@@ -33,8 +33,11 @@ jobs:
       if: ${{ matrix.lint-kind == 'code' }}
       with:
         path: ~/.cache/pre-commit
-        key: pre-commit-${{ env.pythonLocation }}-${{ hashFiles('**/.pre-commit-config.yaml') }}
-
+        key: pre-commit-${{ runner.os }}-python-${{ env.pythonLocation }}-${{ hashFiles('**/.pre-commit-config.yaml') }}
```

Comment:
I don't think this is necessary, as the cache key will be search on the PR's target branch if it's not on the feature branch.

Reply:
did this as a temporary solution as ci's failed. the reformatting was also because of ci for some reason. will undo this and the reformatting